### DISCUSSION TITLE
Cannot get final name for export 'encodeXML' of ./node_modules/entities/lib/esm/index.js

### DISCUSSION THREAD

**Discussion by sage-code82** [vercel/community#5180] [Help]
### Summary

I am very new to coding and when I try do deploy my portfolio I receive the following error : 

Failed to compile.
./node_modules/html-to-text/lib/html-to-text.mjs + 24 modules
Cannot get final name for export 'encodeXML' of ./node_modules/entities/lib/esm/index.js
> Build failed because of webpack errors
Error: Command "npm run build" exited with 1

I have seen a few "fixes" but I'll be honest they all seem to be different answers and not all of them seem to work.  Is there an obvious fix for this? 

### Steps to Reproduce

This is my jsonpackage as I think the problem is to do with that? 

{
  "name": "my-portfolio",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@react-email/components": "^0.0.12",
    "@react-email/tailwind": "^0.0.13",
    "clsx": "^2.0.0",
    "framer-motion": "^10.16.15",
    "next": "14.0.3",
    "react": "^18",
    "react-dom": "^18",
    "react-hot-toast": "^2.4.1",
    "react-icons": "^4.12.0",
    "react-intersection-observer": "^9.5.3",
    "react-vertical-timeline-component": "^3.6.0",
    "resend": "^2.0.0"
  },
  "devDependencies": {
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "@types/react-vertical-timeline-component": "^3.3.6",
    "autoprefixer": "^10.0.1",
    "eslint": "^8",
    "eslint-config-next": "14.0.3",
    "postcss": "^8",
    "tailwindcss": "^3.3.0",
    "typescript": "^5"
  }
}

**Comment by amankr1469** [comment]
(https://github.com/resend/resend-node/issues/256) check this out

**Reply by amankr1469** [reply]
npm i resend@2.1.0-canary.1 this should work for you

**Comment by lvegerano** [comment]
I have the same issue but with cheerio. I also tried @amankr1469 and no dice 😢 

EDIT: Just found this https://github.com/cheeriojs/cheerio/issues/3517
